standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -939,6 +940,16 @@ struct DropPartitionsRequest {
   9: optional string catName
 }
 
+struct DropPartitionRequest {
+  1: optional string dbName,
+  2: optional string tblName,
+  3: optional string partName,
+  4: optional list<string> partVals,
+  5: optional bool deleteData,
+  6: optional EnvironmentContext environmentContext,
+  7: optional string catName
```

Comment:
should the catName be the first in this struct? just looks cleaner. I know it was added much later than the others for some of the other structs. But we are adding a new one.